File: python/q2.py
#!/usr/bin/python3

arr=[1,2]
counter=1
total=0
while arr[counter]<=4000000:
    if arr[counter]%2==0:
        total+=arr[counter]
    arr.append(arr[counter]+arr[counter-1])
    counter+=1
print(total)

File: go/q2.go
 package main
import ("fmt")

func main() {
	arr := []int{1,2}
	var counter int = 1
	var total int = 0
	for arr[counter]<=4000000 {
		if arr[counter]%2==0{
			total += arr[counter]
		}
		arr = append(arr, arr[counter]+arr[counter-1])
		counter+=1
	}
	fmt.Println(total)
}
